# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderMacIOSurfaceTextureHost.h
# Commit: 91d8f0f829fd
# Full Hash: 91d8f0f829fd2ae6ec234a6a3f372bf503ed2fbd
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r 91cb8ee8ddd8 -r 91d8f0f829fd gfx/webrender_bindings/RenderMacIOSurfaceTextureHost.h
--- a/gfx/webrender_bindings/RenderMacIOSurfaceTextureHost.h	Thu Nov 05 19:26:21 2020 +0000
+++ b/gfx/webrender_bindings/RenderMacIOSurfaceTextureHost.h	Thu Nov 05 20:18:13 2020 +0000
@@ -41,7 +41,8 @@
   gfx::SurfaceFormat GetFormat() const override;
   gfx::ColorDepth GetColorDepth() const override;
   gfx::YUVColorSpace GetYUVColorSpace() const override;
-  bool MapPlane(uint8_t aChannelIndex, PlaneInfo& aPlaneInfo) override;
+  bool MapPlane(RenderCompositor* aCompositor, uint8_t aChannelIndex,
+                PlaneInfo& aPlaneInfo) override;
   void UnmapPlanes() override;
 
  private: